add scattererOPM CKAN

AnisDjerrab opened the pull request:
This mod aims to replace the outdated Pood's OPM-VO mod for OPM (see #5354).

- <https://github.com/AnisDjerrab/Scatterer-OPM>

AnisDjerrab commented:
Hello CKAN team,  
It has been 5 days since I submitted my mod, and I haven’t received any feedback yet. It would be greatly appreciate it if you could reference it or let me know if there are any corrections I should make.  

1. Name: Scatterer for OPM  
2. Identifier: ScattererOPM  
3. Version: 1.0.0

Thank you in advance for your time and your amazing work for the community!

HebaruSan commented:
Hi @AnisDjerrab, thanks for the submission.

Reviewing this, I noticed this incorrect usage of the `:FOR[]` clause for ModuleManager:

```
@Scatterer_planetsList:FOR[OPM]
```

This declares that your mod itself _is_ OPM. If someone ever installed your mod without OPM, ModuleManager would still think OPM was installed and enable all patches that say `:NEEDS[OPM]`. You shouldn't use `:FOR[]` here. Depending on your intention, `:NEEDS[OPM]` might make sense.

HebaruSan commented on NetKAN/ScattererOPM.netkan:
```diff
@@ -0,0 +1,18 @@
+license: MIT
+ksp_version_min: 1.12.1
+ksp_version_max: 1.12.5
+resources:
+  homepage: https://forum.kerbalspaceprogram.com/topic/227158-scatterer-for-opm/
```
Hi @AnisDjerrab, one last suggestion:

It would help your non-CKAN users if they could find this link from the GitHub repo itself. Luckily if you click the gear icon in the About section, there's a website field in the popup where you can paste this link. If you do that, CKAN can pull from that field instead of hard-coding it here in the .netkan file. This has the advantage of leaving the link fully under your control, so if you want to change it at any point, you don't even need to submit a pull request, you could just change it in your repo and it would automatically update in CKAN.

AnisDjerrab commented:
Hi @HebaruSan,
Thank you very much for your suggestions,
I have added the forum link to the URL section in "About," and I have replaced FOR[OPM] with NEEDS[OPM]. I have also updated the version to 1.0.1 and switched to the YAML format in my NetKAN file.
Any further corrections?

HebaruSan commented:
Thanks for the submission!

AnisDjerrab commented:
> * ,
I will double-check my commits next time. Thank you!
